assets/_scss/projects/project-container.scss

```diff
@@ -130,20 +130,34 @@
   display: inline;
 }
 
-@media screen and (max-width: 500px) {
+@media screen and (max-width: 768px) {
   .project-container {
     margin-left: unset;
     margin-right: unset;
+    .project-img {
+      width: 100%;
+      text-align: center;
+    }
+    .project-header {
+      width: 100%;
+    }
   }
 }
+@media (max-width: 600px) {
+  .project-img img {
+    height: fit-content !important;
```

Comment:
It's not a good practice to use `!important`, unless is absolutely necessary.

In this case it can be easily prevented.